test/algorithms/buffer/buffer_geo_spheroid.cpp

```diff
@@ -16,6 +18,38 @@ namespace
     std::string const torg = "POINT(10.3937759 63.4302323)";
 }
 
+template
+<
+    typename FormulaPolicy,
+    typename Spheroid,
+    typename CalculationType
+>
+struct geo_buffer_accurate_area
+    : public bg::strategies::buffer::geographic<FormulaPolicy, Spheroid, CalculationType>
+{
+    using base_t = bg::strategies::buffer::geographic<FormulaPolicy, Spheroid, CalculationType>;
+
+public:
+
+    geo_buffer_accurate_area() = default;
+
+    explicit geo_buffer_accurate_area(Spheroid const& spheroid)
+        : base_t(spheroid)
+    {}
+
+    template <typename Geometry>
+    auto area(Geometry const&,
+                std::enable_if_t<! bg::util::is_box<Geometry>::value> * = nullptr) const
+    {
+        return bg::strategy::area::geographic
+            <
+                bg::strategy::karney,
+                bg::strategy::default_order<bg::strategy::karney>::value,
+                Spheroid, CalculationType
+            >(base_t::m_spheroid);
+    }
+};
```

Comment:
Thanks. So it uses karney, whatever the other formula is. Fine for me.
Maybe this option should be documented, it's good to know for users.
But that can be done in another PR.

Reply:
Yes, it needs some further investigation. I will work on it. We know that those formulas (andoyer, thomas, vincenty) are unstable in same corner cases (antipodal points, points very close to each other, meridian/equatorial cases). Those results can be OK/expected for distance or azimuth but if they are used inside other algorithms like area or buffer the results can be unexpected. In general the Geographic part of the library is not documented.